include/SFML/Graphics/Font.hpp

```diff
@@ -95,14 +95,6 @@ class SFML_GRAPHICS_API Font
     ////////////////////////////////////////////////////////////
     Font& operator=(Font&&) noexcept;
 
-    ////////////////////////////////////////////////////////////
-    /// \brief Destructor
-    ///
-    /// Cleans up all the internal resources used by the font
-    ///
-    ////////////////////////////////////////////////////////////
-    ~Font();
-
```

Comment:
A CI job is failing because we removed this declaration. It's hard to explain and I don't totally understand it myself but because we use forward declarations to speed up compilation in other parts of the library, it forces us to declare destructors even if we want them to be default.

Reply:
interesting....let me add this back in then

Reply:
That is because, on Android, `Font` contains a `std::unique_ptr` to `priv::ResourceStream` which is an incomplete type at the moment of `Font` class definition.